src/api-service/__app__/onefuzzlib/azure/vm.py

```diff
@@ -211,7 +211,7 @@ class VM(BaseModel):
     auth: Authentication
 
     def is_deleted(self) -> bool:
-        return has_components(str(self.name))
+        return not has_components(str(self.name))
```

Comment:
While we're in here, maybe it'd be good to add a comment explaining why this check tells us the information we think it does.